charts/gateway-addon/Chart.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v2
```

Comment:
the current dir structure is `gateway-helm` , can we rename the dir here to `gateway-add-ons-helm` which would also represent the chart name in the OCI registry.
reg naming, I'll bring this up in the community meeting today

Reply:
rename to `gateway-addons-helm` by final decision